src/Chainweb/Pact/PactService.hs

```diff
@@ -270,17 +271,19 @@ lookupBlockHeader bhash ctx = do
 -- | Loop forever, serving Pact execution requests and reponses from the queues
 serviceRequests
     :: forall logger tbl. (Logger logger, CanReadablePayloadCas tbl)
-    => LogFunction
+    => logger
```

Comment:
why are we passing in a logger, when there is a logger in the PactServiceEnv?

Reply:
I believe that this is an oversight from my PactService logging PR that this is still there.

Reply:
Fixed this.